.github/workflows/pr_evaluation.yaml

```diff
@@ -0,0 +1,89 @@
+---
+name: PR evaluation
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened, ready_for_review, edited]
+    branches: ['main']
+
+jobs:
+  detect-unresolved-conflicts:
+    name: Detect unresolved merge conflicts
+    runs-on: ubuntu-latest
+    needs: semantic_pr
+    steps:
+      - uses: actions/checkout@v3
+      - name: List files with merge conflict markers
+        run: git --no-pager grep "<<<<<<<" ":(exclude).github/" || true
+      - name: Fail or succeed job if any files with merge conflict markers
+        run: exit $(git grep "<<<<<<<" ":(exclude).github/" | wc --lines)
+
+  pre-commit:
+    name: Pre-commit checks
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v4
+
+      - name: Set up environment
+        uses: ./.github/actions/setup/poetry
+        with:
+          os: ${{ job.os }}
+          python-version: '{{ cookiecutter.python_version }}'
+          poetry-install-options: "--only=misc --no-root"
+          poetry-export-options: "--only=misc"
+
+      - name: Check all the pre-commit hooks passed
```

Comment:
> Check that all pre-commit hooks passed

Reply:
:exploding_head: this is what I have in cookiecutter though!!!! Do I have change it there as well?

Reply:
Why the need for poetry here?

Reply:
Please ignore, was copying the structure from a python project, must have been left in. Will remove.

Reply:
Yes, not a biggie of course. You can change it here, and sneak it in some other time in the cookiecutter template, if you remember

Reply:
Ah, yes, overlooked the Poetry artifacts here - good catch @vemonet